src/NServiceBus.AcceptanceTests/Satellites/When_a_message_is_available.cs

```diff
@@ -56,7 +56,7 @@ protected override void Setup(FeatureConfigurationContext context)
                         {
                             var testContext = builder.Build<Context>();
                             testContext.MessageReceived = true;
-                            testContext.TransportTransactionAddedToContext = ReferenceEquals(pushContext.Context.Get<TransportTransaction>(), pushContext.TransportTransaction);
+                            testContext.TransportTransactionAddedToContext = ReferenceEquals(pushContext.Extensions.Get<TransportTransaction>(), pushContext.TransportTransaction);
```

Comment:
Not sure it makes sense to change in this PR, but we need to go and scrub the remaining "pushContext" usages in the code.